tests/security/permissions/permissions_test_participant_01.xml

```diff
@@ -10,6 +10,7 @@
       </validity>
       <allow_rule>
         <domains>
+          <min>0</min>
```

Comment:
```suggestion
          <id>0</id>
```
there are more of these, and the signed docs need to get updated

Reply:
I think I left this in here by accident when I was trying to get validation working, which I tested by having something wrong. I'll change it to the right think which allows all domains.